src/Status/Api.elm

```diff
@@ -0,0 +1,26 @@
+module Status.Api exposing (getStatus)
+
+import Utils.Api exposing (baseUrl)
+import Http
+import Status.Types exposing (StatusMsg(NewStatus), StatusResponse)
+import Json.Decode exposing (Decoder, map2, string, field, at)
+import Types exposing (Msg(MsgForStatus))
+
+
+getStatus : Cmd Msg
+getStatus =
+    let
+        url =
+            String.join "/" [ baseUrl, "status" ]
+
+        request =
+            Http.get url decodeStatusResponse
```

Comment:
Great pattern! Prefer the ApiResponse monade over the Maybe monade as well. Sorry, haven't payed enough attention to your implementation there.

Later today we need to talk about when to poll new data from the server.